CMakeLists.txt

```diff
@@ -629,6 +629,9 @@ if (LBANN_HAS_OPENCV)
   target_link_libraries(lbann PUBLIC ${OpenCV_LIBRARIES})
 endif ()
 target_link_libraries(lbann PUBLIC ${CONDUIT_LIBRARIES})
+if (Python_Development.Embed_FOUND)
+  target_link_libraries(lbann PUBLIC Python::Python)
```

Comment:
This library is [already linked](https://github.com/LLNL/lbann/blob/develop/CMakeLists.txt#L644). Is it sufficient to just make the `Development.Embed` change above?

Reply:
Hm, I wonder why it wasn't working for me before. `Development.Embed` should be automatically activated when the `Development` module is chosen, so that change was not strictly needed.